lib/OpenQA/WebAPI/Controller/Admin/Workers.pm

```diff
@@ -87,6 +87,7 @@ sub previous_jobs_ajax {
                 push(
                     @data,
                     {
+                        DT_RowId     => "job_" . $job->id,
```

Comment:
I wonder where is it used...

Reply:
in datatables driven by ajax - google for DT_RowId

Reply:
Guessed so :)